spring-cloud-dataflow-docs/src/main/asciidoc/tasks.adoc

```diff
@@ -196,6 +196,19 @@ the sample below postgresql has been chosen.
 1. Save the changed pom.xml
 2. Build the application as described here: <<appendix-building.adoc#building, Building Spring Cloud Data Flow>>
 
+==== Task Application Repository
+
+When launching a task application be sure that the database driver that is being
+used by Spring Cloud Dataflow is also a dependency on the task application.  For
+example if your Spring Cloud Dataflow is set to use Postgresql be sure that
```

Comment:
"Spring Cloud Dataflow" should be "Spring Cloud Data Flow". The same applies in the next section, too.